rosa_backing_up_and_restoring_applications/backing-up-applications.adoc

```diff
@@ -25,7 +25,7 @@ include::modules/oadp-installing-oadp-rosa-sts.adoc[leveloffset=+1]
 
 [id="rosa-backing-up-applications-known-issues"]
 == Known issues
-.Restic is not supported or recommended
+.Restic, Kopia, and DataMover are not supported or recommended.
```

Comment:
```suggestion
.Restic, Kopia, and DataMover are not supported or recommended
```
Do not end a title with a period, per https://www.ibm.com/docs/en/ibm-style?topic=punctuation-periods#headings-and-titles